### PR TITLE
bump 2.8 to unblock 2.8 release

### PR DESCRIPTION
**Pull request by jngz-es**
### Description
bump 2.8 to unblock 2.8 release.
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [ ] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: common/build.gradle [hunk 1]
```diff
@@ -21,8 +21,8 @@ dependencies {
 
 jacocoTestReport {
     reports {
-        xml.enabled false
-        csv.enabled false
+        //xml.enabled false
+        //csv.enabled false
```
Comment (b4sjoo) [inline common/build.gradle:25]: You might need to add a `html.required = true` in reports part.

File: common/build.gradle [hunk 2]
```diff
@@ -45,5 +45,5 @@ jacocoTestCoverageVerification {
 }
 check.dependsOn jacocoTestCoverageVerification
 
-lombok.config['lombok.nonNull.exceptionType'] = 'JDK'
+//lombok.config['lombok.nonNull.exceptionType'] = 'JDK'
```
Comment (b4sjoo) [inline common/build.gradle:48]: You might need to add an additional `lombok.config` file under the project root, maybe you can follow this - https://github.com/opensearch-project/ml-commons/blob/main/lombok.config

File: common/build.gradle (hunk 1 above)
Comment (jngz-es) [reply common/build.gradle:25]: will add, thanks

File: common/build.gradle (hunk 2 above)
Comment (jngz-es) [reply common/build.gradle:48]: will add, thanks

**Review by Zhangxunmt — APPROVED** [review]

**Pull request merged**